pandas/compat/numpy/function.py

```diff
@@ -107,6 +108,12 @@ def validate_argmax_with_skipna(skipna, args, kwargs):
 ARGSORT_DEFAULTS['axis'] = -1
 ARGSORT_DEFAULTS['kind'] = 'quicksort'
 ARGSORT_DEFAULTS['order'] = None
+
+if LooseVersion(_np_version) >= LooseVersion("1.17.0"):
```

Comment:
This is defined in `pandas/compat/numpy/__init__.py`, but I don't want to create a circular import.

Reply:
ok here